build.gradle

```diff
@@ -459,60 +435,37 @@ dependencies {
         exclude group: "net.bytebuddy", module: "byte-buddy"
         exclude group: "net.bytebuddy", module: "byte-buddy-agent"
     }
-    testImplementation ('com.github.hmcts:rd-commons-lib:v0.0.14'){
-        exclude group: 'org.springframework.boot', module: 'spring-boot-starter-web'
-        exclude group: 'org.apache.groovy', module: 'groovy-json'
-        exclude group: 'org.apache.groovy', module: 'groovy'
-        exclude group: 'org.apache.groovy', module: 'groovy-xml'
-    }
-    testImplementation group: 'org.springframework', name: 'spring-test'
-
-    testImplementation group: 'net.bytebuddy', name: 'byte-buddy', version: versions.bytebuddy
-    testImplementation group: 'net.bytebuddy', name: 'byte-buddy-agent', version: versions.bytebuddy
-
-    testImplementation group: 'org.mockito', name: 'mockito-junit-jupiter', version: '4.4.0'
+    testImplementation 'com.github.hmcts:rd-commons-lib:0.1.0'
+    testImplementation group: 'org.mockito', name: 'mockito-junit-jupiter', version: '5.14.2'
 
     testImplementation group: 'org.powermock', name: 'powermock-api-mockito2', version: '2.0.9'
 
+    // tests on version 2 fails on flyway migrations, something like "incorrect syntax"
     testImplementation group: 'com.h2database', name: 'h2', version: '1.4.200'
```

Comment:
Out of scope (tests should not be using h2 anymore, but rather testcontainers.org with postgresql adapter)